<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMessageNotification</class>
 <widget class="QWidget" name="QMessageNotification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QWidget</string>
  </property>
  <widget class="QLabel" name="Notification_title">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>一个亿小目标</string>
   </property>
  </widget>
  <widget class="QLabel" name="Note_date">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>提醒时间到了</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseOK">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>33</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>我知道了</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
